packages/plugins/apollo-usage-report/src/index.ts

```diff
@@ -6,14 +7,16 @@ import {
   YogaLogger,
   type FetchAPI,
 } from 'graphql-yoga';
-import { Report } from '@apollo/usage-reporting-protobuf';
+import { IReportHeader, Report } from '@apollo/usage-reporting-protobuf';
 import {
   ApolloInlineGraphqlTraceContext,
   ApolloInlineRequestTraceContext,
   ApolloInlineTracePluginOptions,
   useApolloInstrumentation,
 } from '@graphql-yoga/plugin-apollo-inline-trace';
 
+const packageJson = JSON.parse(readFileSync('../package.json', 'utf-8'));
```

Comment:
Do you think we can use `import packageJson from '../package.json' with { type: 'json' }` instead ?

```suggestion
import packageJson from '../package.json' with { type: 'json' }
```

Reply:
@EmrysMyrddin that was my first choice but when I run `pnpm check` locally it fails due to this syntax so I tried only `import packageJson from '../package.json'` which was ok for `pnpm check` but it failed here in CI with error that `with { type: 'json' }` argument is missing 🤦 So I ends up with `const packageJson = JSON.parse(readFileSync('../package.json', 'utf-8'))` because only with this syntax the linter had no problem. The linter is not satisfied with this `const packageJson = require('../package.json', 'utf-8')` option either.

@ardatan If you are against importing a specific version of this plugin from `package.json`, then it is probably not a problem to assign only the static name to the `agentVersion` property. e.g.: `agentVersion: '@graphql-yoga/plugin-apollo-usage-report'` or just `agentVersion: '@graphql -yoga'`

Reply:
I added a static value for `agentVersion` property finally and set it to just `graphql-yoga`.

Reply:
I just tested it and everything is working fine. 🙂 
<img width="1021" alt="screenshot 2024-10-31 v 12 37 09 (2)" src="https://github.com/user-attachments/assets/cefaf049-0ca9-4279-934c-0e3915ccd24e">